config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  get 'users/:user_id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:post_id', to: 'posts#show'
+  get 'users', to: 'users#index'
+  get 'users/:user_id', to: 'users#show'
+end
```

Comment:
[OPTIONAL]: You could also practice using `resources` it makes your code clean and readable, like the code below, 

````
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
````

Reply:
Thanks @micheaol 
I wanted to add a comment and ask if I should use resources for this case or not. I forgot. Thank you for your suggestion.

Should I apply it before merge?